haystack/components/preprocessors/document_splitter.py

```diff
@@ -112,6 +114,9 @@ def run(self, documents: List[Document]):
                 raise ValueError(
                     f"DocumentSplitter only works with text documents but content for document ID {doc.id} is None."
                 )
+            if doc.content == "":
```

Comment:
strip?

Reply:
hrmm not sure, that would be changing previous behavior so would technically constitute a breaking change. So I'd rather not. If anything it's odd that the DocumentSplitter was already filtering out Documents with empty strings as content.

Reply:
I can't see any shortcomings with this change - I'm OK with it, the `strip()`, as Sebastian mentioned, would be a change of behaviour.

Reply:
okay 👍🏻